run_pipeline.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
-digicam-raw --compute --output=./ /sst1m/raw/2018/08/19/SST1M_01/SST1M_01_20180819_{001..003}.fits.fz
-# digicam-pipeline --compute --display --output=. --dark=raw_histo.pk --parameters=digicampipe/tests/resources/calibration_20180814.yml /sst1m/raw/2018/08/19/SST1M_01/SST1M_01_20180819_{004..010}.fits.fz
\ No newline at end of file
+# digicam-raw --compute --output=./ /sst1m/raw/2018/08/19/SST1M_01/SST1M_01_20180819_{001..003}.fits.fz
+digicam-pipeline --compute --display --output=. --dark=raw_histo.pk --parameters=digicampipe/tests/resources/calibration_20180814.yml /sst1m/raw/2018/08/19/SST1M_01/SST1M_01_20180819_{009..010}.fits.fz --template=digicampipe/tests/resources/pulse_SST-1M_pixel_0.dat --debug
```

Comment:
for the run_pipeline.sh script, I would let both digicam-raw and digicam-pipeline as both are needed when working from raw data.

Reply:
Ok I change the script. Now I will not update it anymore